### DISCUSSION TITLE
Can any data match when the expression is empty

### DISCUSSION THREAD

**Discussion by huangyedie** [microsoft/RulesEngine#225] [Q&A]
How should the expression be written to match any data

**Comment by alexreich** [comment]
Not sure what you mean here - an expression is evaluated to be true or false.

Do you mean something like `input1 != null` (not null) or `input1 != ""` (not empty string)?

**Comment by abbasc52** [comment]
@huangyedie , if you want an expression to be always true, you can just pass Expression as `"true"`. But i don't recommend it unless it is some default condition
